File: Python/math/ques6(prime_number).py
from math import sqrt
def isPrime(num):
    if num == 2: return True

    flag = 0
    for i in range(2,int(sqrt(num))+1):
        if num%i == 0:
            flag = 1
            break
    return False if flag == 1 else True


num = 14
print(isPrime(num))

File: Javascript/math/ques6(prime_number).js
function isPrime(num){
    if(num === 2) return true

    let flag = 0
    for(let i=2;i*i<=num;i++){
        if(num%i === 0) {
            flag = 1
            break
        }
    }
    return flag === 1 ? false : true
}

let num = 18
console.log(isPrime(num));